src/core/api/checkout/checkout_nodo_ws/v1/_base_policy.xml.tpl

```diff
@@ -9,11 +9,55 @@
     </inbound>
     <outbound>
       <base />
+        
+      <set-variable name="ecommerce_url" value="${ecommerce_url}" />
+      <set-variable name="ccp" value="@((string)context.Request.Body.As<JObject>(preserveContent: true)["soapenv$Envelope"]["soapenv$Body"]["cdInfoWisp"]["codiceContestoPagamento"])" />
+
+      <!-- pagopa-proxy returned error -->
+      <send-request response-variable-name="transaction-check-result" ignore-error="true" timeout="10" mode="new">
+        <set-url>@(String.Format("https://{0}/pagopa-ecommerce-transactions-service/transactions/payment-context-codes/{1}/activation-results", (string)context.Variables["ecommerce_url"], (string)context.Variables["ccp"] ))</set-url>
+        <set-method>POST</set-method>
+        <set-header name="Content-Type" exists-action="override">
+          <value>application/x-www-form-urlencoded</value>
```

Comment:
@MarcoAGreco json not x-www-form-urlencoded

Reply:
Solved with 373fdd6